src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an example in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObj = {};
+
+  sourceString
+    .split(';')
+    .forEach(row => {
+      let [key, value] = row.split(':');
+
+      key = key.replace(/\n| /g, '');
+      value && (value = value.trim());
+
+      // checking if key is exist and adding values in resultObject
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments)